packages/search-types/src/partial-result.model.ts

```diff
@@ -10,8 +10,10 @@ import { Result } from './result/result.model';
  * @public
  */
 export interface PartialResult extends Previewable {
-  /** {@inheritDoc Previewable.results} */
+  // eslint-disable-next-line tsdoc/syntax
+  /** {@inheritDoc Previewable.results}. */
```

Comment:
The problem here is the dot (`.`) at the end of the sentence. This is causing the `eslint` to detect this as a summary instead of just an `@inheritDoc` tag. Disabling the rule `jsdoc/require-description-complete-sentence` instead should solve the issue.

```suggestion
  // eslint-disable-next-line jsdoc/require-description-complete-sentence
  /** {@inheritDoc Previewable.results} */
```